styles/SearchDropdown/dropdown.module.css

```diff
@@ -0,0 +1,81 @@
+.searchBar {
```

Comment:
You haven't made any huge changes in the CSS right??

Once we make a release we'll take some time to clean up / streamline our CSS

Reply:
Yeah, not huge changes